### ISSUE TITLE
[Voice Access] SplitButton secondary button cannot be targeted

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#7818]
### Describe the bug

Voice access has a feature where each interactable control can have a number associated with it for targeting. SplitButtons do not have a number tag present on the secondary button.

### Steps to reproduce the bug

1. Open WinUI 2 Gallery > SplitButton
2. Open Voice Access
3. Say "Show numbers"

### Expected behavior

A number tag should appear on each the secondary button for the split button.

### Screenshots

Unfortunately, summoning snipping tool removes the number tags. However, the accessibility team managed to get a screen capture of the issue in Windows Terminal.

[Voice.Access.users.are.not.able.to.access.More.Options.dropdown.icon.control.using.Show.Numbers.command.zip](https://github.com/microsoft/microsoft-ui-xaml/files/9760394/Voice.Access.users.are.not.able.to.access.More.Options.dropdown.icon.control.using.Show.Numbers.command.zip)

Though Windows Terminal is a XAML Islands app, this issue repros with the WinUI 2 Gallery (a non XAML Islands app).

### Windows app type

- [ ] UWP
- [ ] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (22H2): Build 22621

**Comment by carlos-zamora** [comment]
This is still an issue

**Comment by ranjeshj** [comment]
SplitButton exposes both Invoke and Expand patterns. VoiceAccess needs to show numbers for both which seems to be the issue. Will forward the bug to VoiceAccess team.
